luaui/Include/blueprint_substitution/reports.lua

```diff
@@ -0,0 +1,182 @@
+-- luaui/Include/blueprint_substitution/reports.lua
+-- Contains reporting functions for api_blueprint.lua
+
```

Comment:
preserved these simply because we know the mappings are incomplete, and they were useful for programmatic processing.